Rakefile

```diff
@@ -18,42 +18,44 @@ end
 
 desc "Run Android tests on a device or emulator"
 task :integration_tests do
-  output = `adb devices`
-  if output.match(/device$/)
-      sh "./gradlew --continue connectedAndroidTest"
-  else
-    puts "Please connect a device or start an emulator and try again"
-    exit 1
-  end
+  puts "This project has no integration tests 🙏"
```

Comment:
... is this really what we want to do?

Reply:
Should we just remove the task? Or better yet, add an integration test?

Reply:
I'm game for whichever. This project does indirectly gain integration test coverage from Drop-In. Before, the task would run and not find any tests.

We put in a custom string with an emoji to give a friendlier output when running `rake integration_tests`.

Reply:
My preference would be for this project to have at least one integration test of it's own. But if that's tough to add, I would just delete the task!

Reply:
We've added an integration test for this.